From ec1c729eddd6cf30c3d723461f3f993cd5d31a2e Mon Sep 17 00:00:00 2001
From: Romain Naour <romain.naour@gmail.com>
Date: Sat, 1 Aug 2020 02:25:47 +0200
Subject: [PATCH 01/10] ecore_evas/engines/drm/meson.build: use "gl_deps" as
 engine dependencies

Meson trigger an error since "engine_gl_drm" variable doesn't exist.

src/modules/ecore_evas/engines/drm/meson.build:10:2: ERROR: Unknown variable "engine_gl_drm".

Instead use "gl_deps" as engine dependencies.

Upstream status:
https://sourceforge.net/p/enlightenment/mailman/message/37075016/

Signed-off-by: Romain Naour <romain.naour@gmail.com>
---
 src/modules/ecore_evas/engines/drm/meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/modules/ecore_evas/engines/drm/meson.build b/src/modules/ecore_evas/engines/drm/meson.build
index 5807101..0741d7f 100644
--- a/src/modules/ecore_evas/engines/drm/meson.build
+++ b/src/modules/ecore_evas/engines/drm/meson.build
@@ -3,7 +3,7 @@ engine_deps = [ecore_drm2, ecore_input, eeze, elput, libdrm, engine_drm]
 
 if get_option('opengl') != 'none'
   config_h.set('BUILD_ECORE_EVAS_GL_DRM', '1')
-  engine_deps += [engine_gl_drm]
+  engine_deps += [gl_deps]
 endif
 
 engine_include_dir += include_directories(join_paths('..', '..', '..', 'evas', 'engines', 'drm'))
-- 
2.20.1

